google/devtools/clouddebugger/clouddebugger.yaml

```diff
@@ -0,0 +1,70 @@
+type: google.api.Service
+
+config_version: 2
+
+name: clouddebugger.googleapis.com
+title: Stackdriver Debugger API
+
+documentation:
+  summary: |
+    Examines the call stack and variables of a running application
+    without stopping or slowing it down.
+
+apis:
+- name: google.devtools.clouddebugger.v2.Debugger2
+- name: google.devtools.clouddebugger.v2.Controller2
+
+http:
```

Comment:
Why? It's information we leverage (for idempotency, potentially for identifying required vs optional fields), and we leave it in the proto annotations when the API authors set it that way.